projects/igniteui-angular/src/lib/calendar/calendar.component.html

```diff
@@ -47,13 +47,13 @@ <h2 class="igx-calendar__header-date">
         [@animateChange]="animationAction"
         (@animateChange.done)="animationDone($event)">
         <igx-days-view *ngFor="let view of dayViews; index as i;" [changeDaysView]="true" #days
+                [selection]="selection"
```

Comment:
The 'selection' input sets the `selectedDates` to null and it needs to be called first initially, otherwise overrides the `value` input.